### ISSUE TITLE
better debugging messages

### ISSUE THREAD

**Issue by ahmadnassri** [Mashape/analytics-agent-jquery#1]
feedback from user:

```
I'm getting this error / response from https://socket.analytics.mashape.com/1.0.0/single [{"field":"data.har.log.entries.0.request.url","message":"must be uri format"}]
```

**Comment by nijikokun** [comment]
Not sure how this is an issue with the agent, looks more like an issue with the API, such as what is the URI format?

**Comment by ahmadnassri** [comment]
> Not sure how this is an issue with the agent, looks more like an issue with the API, such as what is the URI format?

the invalid URI format is the problem, but in order for us to debug remotely from users, be good to have debugging flags, or info in the console?

**Comment by nijikokun** [comment]
Hmm, yeah I couldn't decipher that from your initial comment.

If it's something like a console error, that's easy enough. Though the API should return something a little better.

**Comment by ahmadnassri** [comment]
yeah the validator currently just reports the invalid field(s) but not the values
